packages/macro/src/index.macro.ts

```diff
@@ -118,10 +122,10 @@ const macro: MacroHandler = ({ references, state }) => {
         throw new Error(`The file ${filenameNode}(.js/.ts/.jsx/.tsx) does not exist.`);
       }
 
-      const useSSRComputationFunctionName = `useSSRComputation_${side.charAt(0).toUpperCase() + side.slice(1)}`; 
+      const useSSRComputationFunctionName = `useSSRComputation_${side.charAt(0).toUpperCase() + side.slice(1)}`;
       parent.callee = t.identifier(useSSRComputationFunctionName);
-      addImportStatement(useSSRComputationFunctionName, `@popmenu/use-ssr-computation.runtime/lib/${useSSRComputationFunctionName}`, true, nodePath);
-      
+      addImportStatement(useSSRComputationFunctionName, `@popmenu/use-ssr-computation.runtime/lib/${useSSRComputationFunctionName}`, true, false, nodePath);
```

Comment:
convert the statement to `import * as <__computationName> from '......ssr-computation';` to import both `compute` and `execute` functions